SPARQLWrapper/SPARQLExceptions.py

```diff
@@ -46,23 +49,25 @@ class EndPointInternalError(SPARQLWrapperException):
     Exception type for Internal Server Error responses. Usually HTTP response status code ``500``.
     """
 
-    msg = "endpoint returned code 500 and response"
+    msg = "The endpoint returned the HTTP status code 500"
 
 
 class QueryBadFormed(SPARQLWrapperException):
     """
     Query Bad Formed exception. Usually HTTP response status code ``400``.
     """
 
-    msg = "a bad request has been sent to the endpoint, probably the sparql query is bad formed"
+    msg = "A bad request has been sent to the endpoint: it is probably the SPARQL query is badly formed"
 
 
 class EndPointNotFound(SPARQLWrapperException):
     """
     End Point Not Found exception. Usually HTTP response status code ``404``.
     """
 
-    msg = "it was impossible to connect with the endpoint in that address, check if it is correct"
+    msg = (
+        "It was not possible to connect with the endpoint given, check if it is correct"
```

Comment:
given endpoint?

Reply:
sounds good, I would maybe write "It was not possible to connect to the endpoint given, check if it is correct"